modules/apps/export-import/export-import-service/src/main/java/com/liferay/exportimport/internal/background/task/LayoutSetPrototypeImportBackgroundTaskExecutor.java

```diff
@@ -184,6 +189,32 @@ public BackgroundTaskResult execute(BackgroundTask backgroundTask)
 		return BackgroundTaskResult.SUCCESS;
 	}
 
+	@Activate
+	protected void activate(
+			BundleContext bundleContext, Map<String, Object> properties)
+		throws PortalException {
+
+		setBackgroundTaskStatusMessageTranslator(
+			new LayoutExportImportBackgroundTaskStatusMessageTranslator());
+
+		LayoutSetPrototypeSystemConfiguration
+			layoutSetPrototypeSystemConfiguration =
+				ConfigurableUtil.createConfigurable(
```

Comment:
@yuhai This component has configuration, but it has no `@Modified` method.

See https://docs.osgi.org/specification/osgi.cmpn/7.0.0/service.component.html#service.component-modification

> If the modified attribute is not specified, then the component configuration will become unsatisfied if its component properties use a Configuration object and that Configuration object is modified in any way.

So we need a `@Modified` method.